Changelog.md

```diff
@@ -27,6 +27,7 @@
 - Fix bug where rubric grades could not be selected with the return key (#6354)
 - Allow admins to set the number of puma workers and threads from the settings files (#6348)
 - Fix bug where a user who has switched roles could not view the about modal or log out (#6247)
+- Fix bug where emoji annotation options were available even when no text/region was selected (#6192)
```

Comment:
@DanielDervishi I forgot to mention this last time, but the changelog number should be the PR number (so 6384, not 6192)